tests/BenchmarkDotNet.Tests/ConfigParserTests.cs

```diff
@@ -560,5 +560,17 @@ public void InvalidEnvVarAreRecognized()
         {
             Assert.False(ConfigParser.Parse(new[] { "--envVars", "INVALID_NO_SEPARATOR" }, new OutputLogger(Output)).isSuccess);
         }
+
+        [Fact]
+        public void UserCanSpecifyNoForceGCs()
+        {
+            var parsedConfiguration = ConfigParser.Parse(new[] { "--noForcedGCs" }, new OutputLogger(Output));
+            Assert.True(parsedConfiguration.isSuccess);
+
+            foreach (var job in parsedConfiguration.config.GetJobs())
+            {
+                Assert.True(job.Environment.Gc.Force == false);
```

Comment:
nit: you could simplify that

```suggestion
                Assert.False(job.Environment.Gc.Force);
```